doc/source/streaming-analytics.rst

```diff
@@ -38,7 +38,7 @@ These functions correspond to the SQL commands ``SELECT`` and ``WHERE``.
    ...                map(get([1, 2])),
    ...                list)
 
-Note: this uses the `curried`_ versions of ``map`` and ``filter``.
```

Comment:
Removing cross linking because the page is not rendering correctly, as seen below.

![image](https://user-images.githubusercontent.com/17503044/87225635-8bea0a00-c35c-11ea-9bdb-6f67ce40c827.png)